prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical
+Silver Service Taxi class
+"""
+from prac_09.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Specialised version of a Taxi that includes fanciness."""
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness):
+        """Initialise a SilverServiceTaxi instance, based on parent class Taxi."""
+        super().__init__(name, fuel)
+        self.price_per_km = fanciness * self.price_per_km
+
+    def __str__(self):
+        """Return a string like a Taxi but a flagfall is accounted."""
+        return f"{super().__str__()} plus flagfall of ${self.flagfall:.2f}"
+
+    def get_fare(self):
+        """Return the price for the taxi trip."""
+        return self.price_per_km * self.current_fare_distance + self.flagfall
```

Comment:
I feel like you could have used 'super().get_fare() + ...' to cover this, it would better follow DRY and modular principles.